src/l10n/en.po

```diff
@@ -32,6 +32,10 @@ msgstr ""
 "Language-Team: \n"
 "Plural-Forms: \n"
 
+#: 0b0c
+msgid "{0}"
```

Comment:
Where is this translation value used?

Reply:
That translation is caused by line 67
```
                            text: t`${item?.text}`,
```

It seems that I did not use the `{ t } from '@bluecateng/l10n.macro'` correctly, resulting in the language segment not being translated as intended.

I'll remove the translation on that line.